__tests__/useSubscription-test.tsx

```diff
@@ -73,7 +75,7 @@ describe('useSubscription', () => {
     });
 
     function MyComponent({ env }) {
-        function InnerComponent() {
+        function InnerComponent(): any {
```

Comment:
```suggestion
        function InnerComponent(): string {
```

Reply:
this causes the error:
JSX element type 'string' is not a constructor function for JSX elements.